contracts/exchangeIssuance/ExchangeIssuanceLeveraged.sol

```diff
@@ -116,6 +114,9 @@ contract ExchangeIssuanceLeveraged is ReentrancyGuard, FlashLoanReceiverBaseV2,
 
     /* ============ Modifiers ============ */
 
+    // TODO: Review if we really need this check.
+    // In the 0x case we see a lot of people using approveTokens to use the contract with non-index-coop sets
+    // Maybe we should make their life easier and just remove this check? Are there any security implications?
```

Comment:
@ncitron : What do you think about this ? Can we just remove `isSetToken` ? Do you see any security implications ?

Reply:
Yeah I think this is fine to remove. Execution would fail anyway later down the line when it tries to issue/redeem with a non set token